Polyhedron/demo/Polyhedron/Scene_item.cpp

```diff
@@ -181,7 +181,7 @@ void CGAL::Three::Scene_item::selection_changed(bool b)
     setSelected(true);
     d->was_selected = true;
     redraw();
-    QTimer::singleShot(500, [this]{setSelected(false); redraw();});
+    QTimer::singleShot(500, this, SLOT(unSelect()));
```

Comment:
Alternatively, you could have used:
```C++
QTimer::singleShot(500, this, [this]{setSelected(false); redraw();});
```
with three arguments.

See the doc: http://doc.qt.io/qt-5/qtimer.html#singleShot-2